nova/conductor/manager.py

```diff
@@ -296,6 +296,12 @@ def _cold_migrate(self, context, instance, flavor, filter_properties,
         image = utils.get_image_from_system_metadata(
             instance.system_metadata)
 
+        # NOTE(jkulik): We need the instance's current host in at least one
+        # filter to make sure we don't pass vCenter boundaries, i.e. shards
```

Comment:
Oh, one more: I assume this code is bound to be removed again, when we eventually do support this migration?

Otherwise it'd feel awkward for me to mention vCenter here, way out in common code.

Reply:
We will never be able to upstream this code. At least I don't expect it to. Mentioning the vCenter should thus be ok, because we only use VMware.